benchmarks/CMakeLists.txt

```diff
@@ -12,29 +12,31 @@
 # For details, see https://github.com/LLNL/Umpire
 # Please also see the LICENSE file for MIT license.
 ##############################################################################
+set(benchmarks_dir ${UMPIRE_BENCHMARKS_OUTPUT_DIR})
```

Comment:
I don't think renaming the variable here helps much? Why not use it directly.

Reply:
👍